contrib/runners/http_runner/http_runner/runner.yaml

```diff
@@ -24,6 +24,7 @@
     password:
       description: Password required by basic authentication.
       type: string
+      secret: true
```

Comment:
The title still doesn't say it all :)

Reply:
As I understand we forgot to include this in master, while having the fix in `v2.8`

Reply:
That's correct.